website/server/controllers/api-v3/challenges.js

```diff
@@ -168,11 +297,12 @@ api.joinChallenge = {
  * @api {post} /api/v3/challenges/:challengeId/leave Leave a challenge
  * @apiName LeaveChallenge
  * @apiGroup Challenge
- * @apiParam {UUID} challengeId The challenge _id
+ * @apiParam (Path) {UUID} challengeId The challenge _id
  *
  * @apiSuccess {Object} data An empty object
  *
  * @apiUse ChallengeNotFound
+ * @apiUse UserNotFound
```

Comment:
The `keep` parameter should be described in this comment block.

`let keep = req.body.keep === 'remove-all' ? 'remove-all' : 'keep-all';`